docs/content/_docs/query_language.html

```diff
@@ -227,72 +192,104 @@ <h4>Operators</h4>
   </tr>
 
   <tr>
-    <th scope="row">Key Does Not Match</th>
-    <td><pre><code class="language-hql">$key != &lt;a&gt;</code></pre></td>
-    <td><pre><code class="language-json">["not", ["key", &lt;a&gt;]]</code></pre></td>
+    <th scope="row">Tag Does Not Exists</th>
+    <td>Tag foo does not exist.</td>
+    <td><pre><code class="language-hql">!+&lt;foo&gt;</code></pre></td>
+    <td><pre><code class="language-json">["not", ["+", &lt;foo&gt;]]</code></pre></td>
   </tr>
 
   <tr>
-    <th scope="row">Tag Exists</th>
-    <td><pre><code class="language-hql">+&lt;a&gt;</code></pre></td>
-    <td><pre><code class="language-json">["+", &lt;a&gt;]</code></pre></td>
+    <th scope="row">Tag Prefixed With</th>
+    <td>Tag foo is prefixed with bar.</td>
+    <td><pre><code class="language-hql">&lt;foo&gt; ^ &lt;bar&gt;</code></pre></td>
+    <td><pre><code class="language-json">["^", &lt;foo&gt;, &lt;bar&gt;]</code></pre></td>
   </tr>
 
   <tr>
-    <th scope="row">Tag In</th>
-    <td><pre><code class="language-hql">&lt;a&gt; in [&lt;b&gt;, ..]</code></pre></td>
-    <td><pre><code class="language-json">["or", ["=", &lt;a&gt;, &lt;b&gt;], ..]</code></pre></td>
+    <th scope="row">Tag Not Prefixed With</th>
+    <td>Tag foo is not prefixed with bar.</td>
+    <td><pre><code class="language-hql">&lt;foo&gt; !^ &lt;bar&gt;</code></pre></td>
+    <td><pre><code class="language-json">["not", ["^", &lt;foo&gt;, &lt;bar&gt;]]</code></pre></td>
   </tr>
 
   <tr>
-    <th scope="row">Tag Not In</th>
-    <td><pre><code class="language-hql">&lt;a&gt; not in [&lt;b&gt;, ..]</code></pre></td>
-    <td><pre><code class="language-json">["not", ["or", ["=", &lt;a&gt;, &lt;b&gt;], ..]]</code></pre></td>
+    <th scope="row">True</th>
+    <td><pre><code class="language-hql">true</code></pre></td>
+    <td><pre><code class="language-json">true</code></pre></td>
+  </tr>
+
+  <tr>
+    <th scope="row">False</th>
+    <td><pre><code class="language-hql">false</code></pre></td>
+    <td><pre><code class="language-json">false</code></pre></td>
   </tr>
 
   <tr>
     <th scope="row">RegEx On Tag</th>
     <td><pre><code class="language-hql">&lt;a&gt; ~ &lt;b&gt;</code></pre></td>
     <td><pre><code class="language-json">["~", &lt;a&gt;, &lt;b&gt;]</code></pre></td>
   </tr>
+
+  <tr>
+    <th scope="row">Custom Filters</th>
+    <td><pre><code class="language-hql">q &lt;a&gt;</code></pre></td>
+    <td><pre><code class="language-json">["q", &lt;a&gt;]</code></pre></td>
+  </tr>
+
 </table>
 </div>
 
+<h4 id="customfilters">
+  Custom Filters
+  <a class="link-to" href="#customfilters"><span class="glyphicon glyphicon-link"></span></a>
+</h4>
+
+<p>
+  When you need to select series using a more complex syntax than that available through the basic tag suggestions, you can create custom filters. These can take advantage of any of the filters mentioned above, or use the HQL language covered in the next section..
```

Comment:
Sounds good, I would add that it can only use the HQL format of the above filters.

Also looks like an extra trailing period .